src/components/Section.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import ItemBlock from './ItemBlock';
+
+interface SectionItem {
+  image: string;
+  title: string;
+  body: string;
+  linkText?: string;
+  linkPath?: string;
+}
+
+interface SectionProps {
+  title: string;
+  data: SectionItem[];
+  linkText?: string;
+  linkPath?: string;
+}
+
+function Section(props: SectionProps): React.Component {
+  const dataWithCorrectedImagePaths = props.data.map(data => { data.image = `/assets/${data.image}`; return data; });
```

Comment:
```suggestion
  const data = props.data.map(d => d.image = `/assets/${data.image}`);
```

.map returns a new array, so no need to `return data`! 

[MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

Reply:
actually not sure in this case because we are mutating an existing object.. so technically we don't even need the variable

Reply:
Yeah, just changed it to a `forEach`